server/routerlicious/packages/services/src/mongodb.ts

```diff
@@ -499,6 +499,7 @@ export class MongoDbFactory implements core.IDbFactory {
 		);
 		// Need to cast to any before MongoClientOptions due to missing properties in d.ts
 		const options: MongoClientOptions = {
+			directConnection: true,
```

Comment:
Let's be safe, make this value configurable, so we can turn off quickly if we found it brings more issues?